.aicoe-ci.yaml

```diff
@@ -1 +1,9 @@
-check: []
+check:
+  - thoth-build
+build:
+  build-stratergy: Containerfile
+  dockerfile-path: github-glue/Containerfile
```

Comment:
```suggestion
  dockerfile-path: Containerfile
```

Reply:
it does not have to point to a specific containerfile?